src/messages/javascript.js

```diff
@@ -240,6 +240,12 @@ export const KNOWN_LIBRARY = {
   ],
 };
 
+export const UNKNOWN_API = {
+  code: 'UNKNOWN_API',
+  message: _('{{api}} is not support by Firefox'),
```

Comment:
Yeah, i had to do this by creating const in messages for each combination. This isn't going to work. If you could figure this out, it could make some of the messages a lot simpler.